placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/json/AndroidPlacesApiJsonParser.java

```diff
@@ -846,8 +846,8 @@ Place readPlace(JsonReader reader) throws IOException {
                 case "geocode":
                     types.add(PlaceType.GEOCODE);
                     break;
-                default:
-                    reader.skipValue();
+                case "establishment":
```

Comment:
don't remove default case entirely please, we don't control the API

Reply:
Sir kindly tell me , what i do to get the lat and log of selected place

Reply:
Hey @mehmood-usman, this isn't the best forum to ask for help (commenting on a pull request).

That said, there's documentation on how to get lat/lon for locations in our [README](https://github.com/seatgeek/android-PlacesAutocompleteTextView#need-more-details-placedetails)